Community/Tdarr_Plugin_advo_FFMPEG_NVENC_Tiered_MKV_with_rollback.js

```diff
@@ -0,0 +1,563 @@
+
+  if (file.mediaInfo.track[0].extra !== undefined
+    && file.mediaInfo.track[0].extra.ADVOVTRANSCODEDONE !== undefined
+    && file.mediaInfo.track[0].extra.ADVOVTRANSCODEDONE === 'true') {
+    response.processFile = false;
+    response.infoLog += '☑File has already been transcoded! \n';
+    return response;
+  }
+
+  response.infoLog += `☑ Codec: ${videoStream.codec_name} \n`;
+
+  // codec will be checked so it can be transcoded correctly
+  if (videoStream.codec_name.toLowerCase() === 'hevc') {
+    isHevc = true;
+
+    if (bitrateprobe == null || bitrateprobe < 100000) {
+      response.processFile = false;
+      response.infoLog += '☑File is already in hevc! \n';
+      return response;
+    }
+
+    if ((file.video_resolution === '480p' || file.video_resolution === '576p')
+      && (bitrateprobe / 1024) < inputs.hevcBitrateCutoff * 0.5) {
+      response.processFile = false;
+      response.infoLog += '☑File is already in hevc! \n';
+      return response;
+    }
+    if (file.video_resolution === '720p' && (bitrateprobe / 1024) < inputs.hevcBitrateCutoff * 0.75) {
+      response.processFile = false;
+      response.infoLog += '☑File is already in hevc! \n';
+      return response;
+    }
+
+    if (file.video_resolution === '1080p' && (bitrateprobe / 1024) < inputs.hevcBitrateCutoff) {
+      response.processFile = false;
+      response.infoLog += '☑File is already in hevc! \n';
+      return response;
+    }
+    if (file.video_resolution === '4KUHD' && (bitrateprobe / 1024) < inputs.hevcBitrateCutoff * 1.5) {
+      response.processFile = false;
+      response.infoLog += '☑File is already in hevc! \n';
+      return response;
+    }
+
+    response.preset = '-c:v hevc_cuvid';
+  } else if (file.video_codec_name.toLowerCase() === 'h264') {
+    if (bitrateprobe == null || bitrateprobe < 100000) {
+      response.processFile = false;
+      response.infoLog += '☑File is below cutoff already! \n';
+      return response;
+    }
+    if ((file.video_resolution === '480p' || file.video_resolution === '576p')
+      && (bitrateprobe / 1024) < inputs.h264BitrateCutoff * 0.5) {
+      response.processFile = false;
+      response.infoLog += '☑File is below cutoff already! \n';
+      return response;
+    }
+    if (file.video_resolution === '720p' && (bitrateprobe / 1024) < inputs.h264BitrateCutoff * 0.75) {
+      response.processFile = false;
+      response.infoLog += '☑File is below cutoff already! \n';
+      return response;
+    }
+
+    if (file.video_resolution === '1080p' && (bitrateprobe / 1024) < inputs.h264BitrateCutoff) {
+      response.processFile = false;
+      response.infoLog += '☑File is below cutoff already! \n';
+      return response;
+    }
+    if (file.video_resolution === '4KUHD' && (bitrateprobe / 1024) < inputs.h264BitrateCutoff * 1.5) {
+      response.processFile = false;
+      response.infoLog += '☑File is below cutoff already! \n';
+      return response;
+    }
```

Comment:
Messages are different, plus perhaps include the calculated bitrate? Like `☑File bitrate is lower than 4000 Kbps for 4K! \n`

Reply:
Messages for hevc changed. The bitrate is already being printed on lines 303-319.